scriptmodules/supplementary/emulationstation.sh

```diff
@@ -106,15 +107,20 @@ _EOF_
         iniSet "gpu_mem_1024" 256
         iniSet "overscan_scale" 1
     else
+        mkdir -p /usr/share/icons/retropie
+        cp "$scriptdir/scriptmodules/$md_type/$md_id/retropie.svg" "/usr/share/icons/retropie/"
         cat > /usr/share/applications/retropie.desktop << _EOF_
```

Comment:
Yes. /usr/local/share is listed in $XDG_DATA_DIRS.
I will move desktop file and icon there.